source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/testcards/TextBlockStrechInColumnSet.json

```diff
@@ -0,0 +1,34 @@
+{
+    "type": "AdaptiveCard",
+    "body": [
+        {
+            "type": "ColumnSet",
+            "columns": [
+                {
+                    "type": "Column",
+                    "items": [
+                        {
+                            "type": "TextBlock",
+                            "text": "long text long text long text long text long text long text",
+                            "wrap": true
+                        }
+                    ],
+                    "width": "stretch"
+                },
+                {
+                    "type": "Column",
+                    "items": [
```

Comment:
shouldn't these cards be using tabs instead of spaces?

Reply:
thanks

---
In reply to: [303629096](https://github.com/microsoft/AdaptiveCards/pull/3195#discussion_r303629096) [](ancestors = 303629096)